Source/core/JSON.h

```diff
@@ -3225,6 +3214,12 @@ namespace Core {
                 }
             }
 
+        protected:
+            void Reset()
```

Comment:
can we keep Reset Public? This is going to affect clients/plugins that call into Reset().

Reply:
Sure